appeals/web/src/server/appeals/issue-decision/issue-decision.mapper.js

```diff
@@ -0,0 +1,293 @@
+			},
+			items: [
+				{
+					classes: 'govuk-input govuk-date-input__input govuk-input--width-2',
+					name: 'day',
+					value: decisionLetterDay || ''
+				},
+				{
+					classes: 'govuk-input govuk-date-input__input govuk-input--width-2',
+					name: 'month',
+					value: decisionLetterMonth || ''
+				},
+				{
+					classes: 'govuk-input govuk-date-input__input govuk-input--width-4',
+					name: 'year',
+					value: decisionLetterYear || ''
+				}
+			]
+		}
+	};
+
+	return {
+		title,
+		backLinkUrl: `/appeals-service/appeal-details/${appealData.appealId}/issue-decision/decision-letter-date`,
+		backLinkText: 'Back',
+		preHeading: `Appeal ${appealShortReference(appealData.appealReference)}`,
+		heading: title,
+		pageComponents: [selectDateComponent]
+	};
+}
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {Appeal} appealData
+ * @param  {import('./issue-decision.types').InspectorDecisionRequest} session
+ * @returns {Promise<PageContent>}
+ */
+export async function checkAndConfirmPage(request, appealData, session) {
+	const title = 'Check your answers';
+
+	const decisionOutcome = mapDecisionOutcome(session?.outcome);
+	const letterDate = session && session.letterDate ? new Date(session.letterDate) : null;
+	const decisionDateText = letterDate
+		? `${letterDate.getDate()} ${letterDate.toLocaleString('default', {
+				month: 'long'
+		  })} ${letterDate.getFullYear()}`
+		: '';
+
+	var documentName, documentURI;
+	if (session.documentId) {
+		const fileInfo = await getFileInfo(
+			request.apiClient,
+			appealData.appealId.toString(),
+			session.documentId
+		);
+		documentName = fileInfo?.latestDocumentVersion.fileName;
+		documentURI = fileInfo?.latestDocumentVersion.documentURI;
+	}
+
+	/** @type {PageComponent} */
+	const summaryListComponent = {
+		type: 'summary-list',
+		parameters: {
+			rows: [
+				{
+					key: {
+						text: 'Decision'
+					},
+					value: {
+						text: decisionOutcome
+					},
+					actions: {
+						items: [
+							{
+								text: 'Change',
+								href: `/appeals-service/appeal-details/${appealData.appealId}/issue-decision/decision`
+							}
+						]
+					}
+				},
+				{
+					key: {
+						text: 'Decision letter'
+					},
+					value: {
+						html: `<a href="${documentURI}" class="govuk-link">${documentName}</a>`
+					},
+					actions: {
+						items: [
+							{
+								text: 'Change',
+								href: `/appeals-service/appeal-details/${appealData.appealId}/issue-decision/decision`
+							}
+						]
+					}
+				},
+				{
+					key: {
+						text: 'Decision date'
+					},
+					value: {
+						text: decisionDateText
+					},
+					actions: {
+						items: [
+							{
+								text: 'Change',
+								href: `/appeals-service/appeal-details/${appealData.appealId}/issue-decision/decision-letter-date`
+							}
+						]
+					}
+				}
+			]
+		}
+	};
+
+	/** @type {PageComponent} */
+	const insetHtmlComponent = {
+		type: 'html',
```

Comment:
This should ideally use the `WarningTextPageComponent` component type instead of html. The whole page component system is quite complex so feel free to ask if you have any questions!